src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -65,7 +96,19 @@ protected void onCreate(Bundle state, boolean ready) {
     int appMessageId = b.getInt("appMessageId");
 
     this.dcContext = DcHelper.getContext(getApplicationContext());
+    if (dcContext.getAccountId() != b.getInt("accountId")) {
+      Toast.makeText(this, "Switch to belonging account first.", Toast.LENGTH_LONG).show();
```

Comment:
- s/belonging/associated/
- I guess it's not easily possible to make it automatically switch to the correct account?

Reply:
automatically switching to the game's account would be cool

Reply:
it displays a weird dummy chat, that is a bit ugly

Reply:
i think, both of "belonging" and "associated" are fine.

however, switching to the account is the nice way to go - with the extra advantage that we do not need the string then :)

but that is sth. for another pr.